src/checks.py

```diff
@@ -243,12 +243,14 @@ def _check_dircmp_only_either_allowed(diff: filecmp.dircmp) -> List[str]:
     allowed_left_only = [
         ".git",
         ".gitignore",
+        ".gitattributes",
+        ".travis.yml",
         ".mvn",
         "mvnw",
         "mvnw.cmd",
         "Jenkinsfile",
     ]
-    allowed_right_only: List[str] = []
+    allowed_right_only: List[str] = ["DEPENDENCIES"]
```

Comment:
fyi in a source-only package like zipkin-api.. there are no dependencies except the implicit framework that executes the compiler. (ex we recently removed a google library dep from zipkin-api). Not sure it will make any difference, here, just noting.